docker/build-and-push.sh

```diff
@@ -13,18 +15,17 @@ project_root=$(git rev-parse --show-toplevel)
 cd "${project_root}"
 
 # Version X.Y.Z of Marquez image to build
-version=$1
+version="$1"
```

Comment:
"${1}"
Not a big deal here, but in other places will keep you out of trouble if you have more than 10 arguments :)